R/mod_body_coc_competition.R

```diff
@@ -0,0 +1,475 @@
+#' body_coc_competition UI Function
+#'
+#' @description A shiny Module.
+#'
+#' @param id,input,output,session Internal parameters for {shiny}.
+#'
+#' @noRd 
+#'
+#' @importFrom shiny NS tagList 
+mod_body_coc_competition_ui <- function(id){
+  ns <- NS(id)
+  tagList(
+    ui_picker_program(
+          inputId = ns("pe_provider"),
+          label = "Select your CoC-funded Provider",
+          choices = sort(pe_summary_validation()$AltProjectName) |> unique(),
+          selected = pe_summary_validation()$AltProjectName[1],
+          options = shinyWidgets::pickerOptions(liveSearch = TRUE,
+                                  liveSearchStyle = 'contains')
+        ),
+    ui_row(
+      title = "Score Summary",
+      DT::dataTableOutput(ns("pe_ProjectSummary")),
+      status = "info",
+      solidHeader = TRUE,
+      collapsible = TRUE
+      ),
+    ui_row(
+          inputId = ns("tabs"),
+          # title = "Client Detail",
+          tabPanel(
```

Comment:
`tabPanel` isn't necessary if you're not building a sidebar. If you want this datatable on it's own row, without a box around it, you can use `ui_row(box = FALSE`.
This applies to all the below datatables